setup.py

```diff
@@ -1,6 +1,8 @@
 from setuptools import setup
+from os.path import expanduser
+home = expanduser("~")
 
-datafiles = [('//etc', ['general_conf/bck.conf'])]
+datafiles = [(home+'/.autoxtrabackup', ['general_conf/bck.conf'])]
```

Comment:
you should better import `os.path` and, call `os.path.join` with `home` and `.autoxtrabackup` folders.

`os.path.join(home, '.autoxtrabackup')`

always let the language to do the hard work :)

Reply:
Thanks for suggestion